audit-ci.json

```diff
@@ -0,0 +1,6 @@
+{
+  "low": true,
+  "allowlist": [
+    1523
```

Comment:
can we add a wee comment and maybe a link to the vulnerability here?

Reply:
It's JSON so I don't think so, but it scan be identified either via https://www.npmjs.com/advisories/ or by referring back to this PR.